userdocs/docs/grpc/types/proto.md

```diff
@@ -0,0 +1,388 @@
+# Protocol Documentation
+<a name="top"></a>
+
+## Table of Contents
+
+- [types/microvm.proto](#types/microvm.proto)
+    - [ContainerVolumeSource](#flintlock.types.ContainerVolumeSource)
+    - [Initrd](#flintlock.types.Initrd)
+    - [Kernel](#flintlock.types.Kernel)
+    - [MicroVM](#flintlock.types.MicroVM)
+    - [MicroVMSpec](#flintlock.types.MicroVMSpec)
+    - [MicroVMSpec.LabelsEntry](#flintlock.types.MicroVMSpec.LabelsEntry)
+    - [MicroVMSpec.MetadataEntry](#flintlock.types.MicroVMSpec.MetadataEntry)
+    - [MicroVMStatus](#flintlock.types.MicroVMStatus)
+    - [MicroVMStatus.NetworkInterfacesEntry](#flintlock.types.MicroVMStatus.NetworkInterfacesEntry)
+    - [MicroVMStatus.VolumesEntry](#flintlock.types.MicroVMStatus.VolumesEntry)
+    - [Mount](#flintlock.types.Mount)
+    - [NetworkInterface](#flintlock.types.NetworkInterface)
+    - [NetworkInterfaceStatus](#flintlock.types.NetworkInterfaceStatus)
+    - [Volume](#flintlock.types.Volume)
+    - [VolumeSource](#flintlock.types.VolumeSource)
+    - [VolumeStatus](#flintlock.types.VolumeStatus)
+  
+    - [MicroVMStatus.MicroVMState](#flintlock.types.MicroVMStatus.MicroVMState)
+    - [Mount.MountType](#flintlock.types.Mount.MountType)
+    - [NetworkInterface.IfaceType](#flintlock.types.NetworkInterface.IfaceType)
+  
+- [Scalar Value Types](#scalar-value-types)
+
+
+
+<a name="types/microvm.proto"></a>
+<p align="right"><a href="#top">Top</a></p>
+
+## types/microvm.proto
+
+
+
+<a name="flintlock.types.ContainerVolumeSource"></a>
+
+### ContainerVolumeSource
+ContainerVolumeSource represents the details of a volume coming from a OCI image.
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| image | [string](#string) |  | Image specifies teh conatiner image to use for the volume. |
```

Comment:
```suggestion
| image | [string](#string) |  | Image specifies the conatiner image to use for the volume. |
```

Reply:
Updated `api/types/microvm.proto` for this.